styles.css

```diff
@@ -66,3 +68,462 @@ body {
   left: 0;
   top: 472px;
 }
+
+.projects {
+  position: absolute;
+  width: 100%;
+  height: 50px;
+  left: 16px;
+  top: 630px;
+  font-family: "Playfair Display", sans-serif;
+  font-style: bold;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.projImg {
+  position: absolute;
+  width: 320%;
+  height: 186px;
+  left: 0;
+  top: 754px;
+  background: url("./images/project1.png");
+  background-repeat: no-repeat;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+}
+
+.container {
+  position: absolute;
+  width: 320px;
+  height: 180px;
+  left: 0;
+  top: 940px;
+  background: #1a2236;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(1px);
+}
```

Comment:
[OPTIONAL]
- [ ] For each project, the content of the project should be aligned with the flexbox. I suggest you fix that. This will give you more options to align its content plus it is responsive.